pyvo/mivot/writer/annotations.py

```diff
@@ -0,0 +1,342 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+MivotAnnotations: A utility module to build and manage MIVOT annotations.
+"""
+import os
+import logging
+
+try:
+    import xmlschema
+except ImportError:
+    xmlschema = None
+# Use defusedxml only if already present in order to avoid a new depency.
+try:
+    from defusedxml import ElementTree as etree
+except ImportError:
+    from xml.etree import ElementTree as etree
+from astropy.io.votable.tree import VOTableFile, Resource
+try:
+    from astropy.io.votable.tree import MivotBlock
+except ImportError:
+    pass
```

Comment:
maybe add a comment here that this try/except should go away once we don't need to do check_astropy_version and require astropy >=6.0